server/src/main/scala/org/scassandra/server/priming/json/ResultJsonRepresentation.scala

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Are you ok with the package change?  Only other thing I see that could be worth moving is PrimingJsonImplicits.   Can move that if it makes sense.